test_crates/trait_associated_const_added/new/src/lib.rs

```diff
@@ -5,30 +5,44 @@ mod sealed {
 // trigger
 pub trait WillGainConstWithoutDefault {
     const BAR: bool;
+
+    fn make_me_object_safe() -> Self;
```

Comment:
These are actually making the trait _stop_ being object-safe, just like having a `const` does.

Perhaps a better name:
```suggestion
    fn make_me_non_object_safe() -> Self;
```